aspnetcore/fundamentals/hosting.md

```diff
@@ -813,8 +828,8 @@ The [IApplicationLifetime interface](/aspnet/core/api/microsoft.aspnetcore.hosti
 | Cancellation Token    | Triggered when&#8230; |
 | --------------------- | --------------------- |
 | `ApplicationStarted`  | The host has fully started. |
-| `ApplicationStopping` | The host is performing a graceful shutdown. Requests may still be processing. Shutdown blocks until this event completes. |
-| `ApplicationStopped`  | The host is completing a graceful shutdown. All requests should be completely processed. Shutdown blocks until this event completes. |
+| `ApplicationStopping` | The host is performing a graceful shutdown. Requests may still be processing. Shut down blocks until this event completes. |
```

Comment:
"shut down" ... on the verb usage above (Line 692). :+1:

This one and the one on Line 832: Shouldn't the noun form (a time) remain one word? "Shutdown"